frontend/src/main/components/Commons/CommonsOverview.js

```diff
@@ -3,11 +3,14 @@ import { Row, Card, Col, Button } from "react-bootstrap";
 import { useNavigate } from "react-router-dom";
 import { hasRole } from "main/utils/currentUser";
 import { daysSinceTimestamp } from "main/utils/dateUtils";
+import PagedAnnouncementTable from "main/components/Commons/PagedAnnouncementTable";
 
 export default function CommonsOverview({ commonsPlus, currentUser }) {
 
     let navigate = useNavigate();
     // Stryker disable next-line all
+    const [isTableVisible, setIsTableVisible] = React.useState(true);
```

Comment:
The `Stryker` comments is there originally, and I just add another code after it without noticing it. Please let me know if I need to write test for these two line